woocommerce/woocommerce-android
# [Tablet support M3] Full flow

## [Tablet support M3] Full flow

**⚠️ Do not merge this PR, as it serves as the foundational PR for subsequent Order list and Order detail tablet support related PRs.**

* https://github.com/woocommerce/woocommerce-android/pull/10694
* https://github.com/woocommerce/woocommerce-android/pull/10763
* https://github.com/woocommerce/woocommerce-android/pull/10959

Closes: https://github.com/woocommerce/woocommerce-android/issues/10614


### Description

This PR adds `NavHostFragment` to the order creation/edition layout. The purpose of the new fragment is to display a two-pane layout on large screens with product selectors on the left and order creation form on the right.

### Testing instructions

1. Run the app on physical tablet or in a tablet emulator
2. Click on the Orders tab and create a new order
3. Observe the two-pane layout is present and it adjusts to screen size (e.g. when rotating the screen, or opening split mode)

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4527432/940ad780-4467-4741-824b-5d1f7226b0f7
